dice.js

```diff
@@ -0,0 +1,16 @@
+const rollDice = function (){
+    const Dice1 = math.floor(math.random()*(6)+1);
+    const Dice2 = math.floor(math.random()*(6)+1);
+
+    return Dice1+Dice2
+};
+Console.log (rollDice());
+ // BOUNS!
+
+ const rollDice = function (more){
+     let result = null
```

Comment:
when you initialize a variable and you want to use it as a counter or a sum, assign it with the value `0`,  
like this : `let result = 0` , with this you can do mathematical operation on the variable without getting unwanted errors or behavior.